src/axom/mint/tests/mint_execution_cell_traversals.cpp

```diff
@@ -379,12 +379,12 @@ AXOM_CUDA_TEST( mint_execution_cell_traversals, for_all_cells_nodeids )
 #if defined(AXOM_USE_RAJA) && defined(AXOM_USE_CUDA) && \
     defined(RAJA_ENABLE_CUDA) && defined(AXOM_USE_UMPIRE)
 
-    const int UnifiedAllocatorID =
-        axom::getResourceAllocatorID( umpire::resource::Unified );
-    const umpire::Allocator prev_allocator = axom::getDefaultAllocator();
-    axom::setDefaultAllocator( axom::getAllocator( UnifiedAllocatorID ) );
-
     using cuda_exec = axom::CUDA_EXEC< 512 >;
+
+    const int exec_space_id  = axom::execution_space<cuda_exec>::allocatorID();
```

Comment:
Instead of having the test routine explicitly request the `umpire::resource::Unified` allocator ID, I see that now you use the fact that `execution_space< axom::CUDA_EXEC<whatever> >::allocatorID()` always returns `umpire::resource::Unified`.

Edit: To clarify, thanks: this is a good simplification.

Reply:
Yes -- our `execution_space< exec_policy >` traits class binds to a default allocator used for that execution space.